src/importer.py

```diff
@@ -0,0 +1,75 @@
+import logging
+import sys
+import time
+
+import settings
+from database import NPRDatabase, LocalDatabase
+from object_store import ObjectStore
+from util import filter_batch_names
+
+
+class Importer:
+    logger = logging.getLogger(__name__)
+    logger.setLevel(logging.INFO)
+
+    logging.basicConfig(
+        stream=sys.stdout,
+        level=logging.WARNING,
+        format="%(asctime)s - %(name)s - %(levelname)s - %(message)s",
+    )
+
+    def __init__(self):
+        super().__init__()
+        self.local_db = LocalDatabase()
+        self.npr_db = NPRDatabase()
+        self.object_store = ObjectStore()
+
+    def import_orphans(self):
+        batch_names = [settings.NPR_NULL_VALUE]
+        self._run_import(batch_names)
+
+    def import_range(self, start_date, num_days_to_import):
+        self.logger.info("Determining batch names for import...")
+        batch_names = self.get_batch_names_for_download(start_date, num_days_to_import)
+        self.logger.info(f"{len(batch_names)} batches found to import")
+        self._run_import(batch_names)
+
+    def _run_import(self, batch_names):
```

Comment:
add `def import_last_x_days()` which calls `import_range()` and imports x days up to yesterday.

Reply:
also ensure we never import today, because that might lead to imcomplete data.

Reply:
add an override flag to indicate if existing results need to be overwritten.